client/README.md

```diff
@@ -142,12 +142,19 @@ func writePoints(clnt client.Client) {
 			"busy": 100.0 - idle,
 		}
 
-		bp.AddPoint(client.NewPoint(
+		pt, err := client.NewPoint(
 			"cpu_usage",
 			tags,
 			fields,
 			time.Now(),
-		))
+		)
+		
+		if err != nil {
+			log.Fatalln("Error: ", err)
```

Comment:
Yes this bothers me also, I just made it to be consistent with previous error handlings in the same sniplet. Let me clear it.